app/assets/javascripts/application.js

```diff
@@ -20,8 +20,18 @@ $(document).ready(function(){
     alert("You must be logged in to check out!")
   });
 
-  // $("#footer").exists(function() {
-  //   $(".container").css({padding-bottom: "150px"})
-  // });
-  //
+  $("input[type=checkbox]").click(function(){
+    $('.categories').hide();
+    var $selected = $('input[type=checkbox]:checked').map(function (index, element) {
+      return '.' + $(element).val()
+    });
+    var categories = $.makeArray($selected).join(',');
+    categories ? $(categories).show() : $('.categories').show();
```

Comment:
Mixed double and single quotes.<br>Expected an assignment or function call and instead saw an expression.